src/View/Home/signup.html.twig

```diff
@@ -3,63 +3,79 @@
 {% block content %}
 	<div class="container sign-up-page">
 		<h2>Page d'inscription</h2>
-		<article>
-			<header>
-				<h3>Vous recevez des musiciens</h3>
-			</header>
-			<form action="" method="post">
-				<h4>Espace d'inscription hôte</h4>
-				<fieldset>
-					<label for="host-name">Votre identifiant :
+		<div class="choose-form">
+			<button id="btn1">Hôte</button>
+			<button id="btn2">Musicien</button>
+		</div>
+		<div id="form-container">
+				<form action="" method="post" id="form1" style="display: none;">
```

Comment:
style="display: none;
This should be in your css file, not here (same for the other form)

Reply:
done